GitHubComments.ps1

```diff
@@ -178,7 +179,8 @@ function Get-GitHubRepositoryComment
     $OwnerName = $elements.ownerName
     $RepositoryName = $elements.repositoryName
 
-    if ($Since -ne $null) {
+    if ($null -ne $Since) 
+    {
         $SinceFormattedTime = $Since.ToUniversalTime().ToString('o')
```

Comment:
I still think you should align on how this is handled in [GitHubIssues](https://github.com/PowerShell/PowerShellForGitHub/blob/master/GitHubIssues.ps1#L233) by not sending the parameter at all if no value has been specified. The API doesn't provide guarantees for how it would treat that type of input, so we should be crisp in the data that we do/do not send.